### PR TITLE
feat: sdk signing support

### PR DESCRIPTION
**Pull request by IloveJavaa**

### CODE REVIEW

File: android/src/main/java/com/reactlibrary/TrackierSDK.java [hunk 1]
```diff
@@ -23,10 +28,17 @@ public String getName() {
...
 		com.trackier.sdk.TrackierSDKConfig sdkConfig = new com.trackier.sdk.TrackierSDKConfig(getReactApplicationContext(), initializeMap.getString("appToken"), initializeMap.getString("environment"));
 		sdkConfig.setSDKType("react_native_sdk");
-		sdkConfig.setSDKVersion("1.6.29");
+		sdkConfig.setSDKVersion("1.6.30");
+		sdkConfig.setAppSecret(secretId,secretKey);
 		com.trackier.sdk.TrackierSDK.initialize(sdkConfig);
 	}
 
+	@ReactMethod
+	public void getAppSecretKey(ReadableMap readableMap){
```
Comment (Hemant-Mann) [inline android/src/main/java/com/reactlibrary/TrackierSDK.java:37]: method name should be setAppSecret

File: index.js [hunk 2]
```diff
@@ -17,6 +17,10 @@ TrackierSDK.initialize = function(trackierConfig) {
 	module_trackier.initializeSDK(trackierConfig)
 }
 
+TrackierSDK.setAppSecret = function(secretId,secretKey){
```
Comment (Hemant-Mann) [inline index.js:20]:
```suggestion
TrackierSDK.setAppSecret = function(secretId, secretKey) {
```

File: index.js [hunk 3]
```diff
@@ -17,6 +17,10 @@ TrackierSDK.initialize = function(trackierConfig) {
...
+	module_trackier.setAppSecretKey({"secretId":secretId,"secretKey":secretKey})
```
Comment (Hemant-Mann) [inline index.js:21]:
```suggestion
	module_trackier.setAppSecret({ "secretId": secretId, "secretKey": secretKey })
```

File: ios/RNTrackierSDK.m [hunk 4]
```diff
@@ -12,6 +12,8 @@ @interface RCT_EXTERN_REMAP_MODULE(TrackierSDK, RNTrackierSDK, NSObject)
  
 RCT_EXTERN_METHOD(initializeSDK:(NSDictionary *)dict)
 
+RCT_EXTERN_METHOD(setAppSecretKey:(NSDictionary *)dict)
```
Comment (Hemant-Mann) [inline ios/RNTrackierSDK.m:15]:
```suggestion
RCT_EXTERN_METHOD(setAppSecret:(NSDictionary *)dict)
```

File: ios/RNTrackierSDK.swift [hunk 5]
```diff
@@ -16,10 +16,16 @@ class RNTrackierSDK: NSObject {
...
 		let config = TrackierSDKConfig(appToken: appToken , env: environment)
 		config.setSDKType(sdkType: "react_native_sdk")
-		config.setSDKVersion(sdkVersion: "1.6.29")
+		config.setSDKVersion(sdkVersion: "1.6.30")
+		config.setAppSecret(secretId: secretId, secretKey: secretKey)
 		TrackierSDK.initialize(config: config)
 	}
 
+	 @objc func setAppSecretKey(_ dict: NSDictionary) -> Void {
+        secretId = dict["secretId"] as! String;
```
Comment (Hemant-Mann) [inline ios/RNTrackierSDK.swift:25]: indentation @IloveJavaa

Comment (Hemant-Mann) [inline ios/RNTrackierSDK.swift:25]: these variables are not set anywhere!

File: ios/RNTrackierSDK.swift [hunk 6]
```diff
@@ -16,10 +16,16 @@ class RNTrackierSDK: NSObject {
 		let environment = dict["environment"] as! String;
 		let config = TrackierSDKConfig(appToken: appToken , env: environment)
 		config.setSDKType(sdkType: "react_native_sdk")
-		config.setSDKVersion(sdkVersion: "1.6.29")
+		config.setSDKVersion(sdkVersion: "1.6.30")
+		config.setAppSecret(secretId: secretId, secretKey: secretKey)
 		TrackierSDK.initialize(config: config)
 	}
 
+	 @objc func setAppSecretKey(_ dict: NSDictionary) -> Void {
```
Comment (Hemant-Mann) [inline ios/RNTrackierSDK.swift:24]:
```suggestion
	 @objc func setAppSecret(_ dict: NSDictionary) -> Void {
```

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/reactlibrary/TrackierSDK.java (hunk 1 above)
Comment (IloveJavaa) [reply android/src/main/java/com/reactlibrary/TrackierSDK.java:37]: fixed

File: index.js (hunk 3 above)
Comment (IloveJavaa) [reply index.js:21]: Done

File: ios/RNTrackierSDK.m (hunk 4 above)
Comment (IloveJavaa) [reply ios/RNTrackierSDK.m:15]: Done

File: ios/RNTrackierSDK.swift (hunk 5 above)
Comment (IloveJavaa) [reply ios/RNTrackierSDK.swift:25]: Done

Comment (IloveJavaa) [reply ios/RNTrackierSDK.swift:25]: Added Variable

File: android/src/main/java/com/reactlibrary/TrackierSDK.java [hunk 7]
```diff
@@ -1,5 +1,7 @@
 package com.reactlibrary;
 
+import android.util.Log;
```
Comment (Hemant-Mann) [inline android/src/main/java/com/reactlibrary/TrackierSDK.java:3]: remove this

File: ios/RNTrackierSDK.swift [hunk 8]
```diff
@@ -11,15 +11,24 @@ import trackier_ios_sdk
 @objc(RNTrackierSDK)
 class RNTrackierSDK: NSObject {
 
+	var secretId: String = ""
+    var secretKey: String = ""
```
Comment (Hemant-Mann) [inline ios/RNTrackierSDK.swift:15]: indentation

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/reactlibrary/TrackierSDK.java (hunk 7 above)
Comment (IloveJavaa) [reply android/src/main/java/com/reactlibrary/TrackierSDK.java]: Done

**Review by Hemant-Mann — APPROVED** [review]

**Pull request merged**